### PR TITLE
[Cloud] Change update and upgrade procedures

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Replaces https://github.com/centreon/centreon-documentation/pull/3017 (renamed branch).

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Review by kduret — COMMENTED** [review]
`poller-update.md` should be removed
If poller is already installed, the script will always try to upgrade to the last major version and update to last minor version of this major

**Comment by jdelpierre** [comment]
Hello @cg-tw 
it's not possible to merge the menu "update" & "upgrade" on a only one single page ? Because it's the same content.

File: cloud/installation/poller-update-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* If your poller is in another major version than the latest, this procedure will not update it to the latest minor version of this major version. The poller will be upgraded to the latest minor version of the latest major version available instead (e.g. it will be upgraded from 23.04.x to 23.10.y).
+
+## Updating or upgrading a poller
+
+1. On your central server, click the arrow next to **Pollers** at the left of the header bar.
+2. In the pop-up that appears, click **Copy install command**. The command is copied to your clipboard.
+3. On your poller, in your terminal, paste the install command and execute it.
+4. [Deploy the poller's configuration](../monitoring/monitoring-servers/deploying-a-configuration.md),
+and choose the **Restart** method for the Engine process.
+5. Finally, restart the Gorgone service if it is used on the poller:
```
Comment (kduret) [inline cloud/installation/poller-update-upgrade.md:21]: For consistency with french doc
```suggestion
1. On your central server, click the arrow next to **Pollers** at the left of the header bar.

2. In the pop-up that appears, click **Copy install command**. The command is copied to your clipboard.

3. On your poller, in your terminal, paste the install command and execute it.

4. [Deploy the poller's configuration](../monitoring/monitoring-servers/deploying-a-configuration.md),
and choose the **Restart** method for the Engine process.

5. Finally, restart the Gorgone service if it is used on the poller:
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/installation/poller-update-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+3. Sur le collecteur, dans le terminal, copiez la commande d'installation et exécutez-la.
+
+4. [Déployez la configuration du collecteur](../monitoring/monitoring-servers/deploying-a-configuration.md), 
+en choisissant la méthode **Redémarrer** pour le processus Engine.
+
+5. Redémarrez enfin le service Gorgone s'il est utilisé sur le collecteur :
+
+  ```shell
+  systemctl restart centengine gorgoned
```
Comment (kduret) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/installation/poller-update-upgrade.md:28]:
```suggestion
  systemctl restart gorgoned
```

File: cloud/installation/poller-update-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  ```shell
+  systemctl restart centengine gorgoned
```
Comment (kduret) [inline cloud/installation/poller-update-upgrade.md:24]:
```suggestion
  systemctl restart gorgoned
```

**Review by kduret — DISMISSED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**